src/number_theory/bernoulli_polynomials.lean

```diff
@@ -176,4 +176,58 @@ begin
   rw [mul_assoc, div_eq_mul_inv, ← mul_inv'],
 end
 
+/-- The following two lemmas can probably done much easier somehow?!-/
```

Comment:
xref https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there.20code.20for.20X.3F/topic/poly.20%28a%29eval.20and.20algebra.20maps/near/229859076

Reply:
Thanks!